db/seeds.rb

```diff
@@ -41,14 +43,31 @@
   'Townhouse Apts'       => 30
 }
 
+# if this array changes then the logic below has to also change
+directions = ['North', 'South']
+
 stops.each do |route_number, stop_names|
-  stop_names.each do |stop_name|
-    # Anytime in the last two months
-    Timecop.freeze rand(86_400).minutes.ago do
-      stop = BusStop.find_or_initialize_by name: stop_name
-      stop.hastus_id = hastus_ids.fetch(stop_name)
-      stop.routes << routes.fetch(route_number)
-      stop.save!
+  directions.each do |direction|
+    if direction == 'North'
+      sequence = 0
+    else
+      sequence = stop_names.length
+    end
+    stop_names.each do |stop_name|
+      if direction == 'North'
+        sequence += 1
+      else
+        sequence -= 1
+      end
+      # Anytime in the last two months
+      Timecop.freeze rand(86_400).minutes.ago do
```

Comment:
Didn't I redo this to actually read two months? Or is that somewhere not merged?

Reply:
That's in the `rewrite` branch.